src/status_im2/contexts/chat/composer/view.cljs

```diff
@@ -62,50 +63,54 @@
+      (:images subs)
+      (:link-previews? subs)
+      (:reply subs)
       (:edit subs)]
-     [gesture/gesture-detector
-      {:gesture (drag-gesture/drag-gesture props state animations dimensions keyboard-shown)}
-      [reanimated/view
-       {:style     (style/sheet-container insets state animations)
-        :on-layout #(handler/layout % state blur-height)}
-       [sub-view/bar]
-       [reply/view state]
-       [edit/view state]
-       [reanimated/touchable-opacity
-        {:active-opacity      1
-         :on-press            (when @(:input-ref props) #(.focus ^js @(:input-ref props)))
-         :style               (style/input-container (:height animations) max-height)
-         :accessibility-label :message-input-container}
-        [rn/selectable-text-input
-         {:ref        #(reset! (:selectable-input-ref props) %)
-          :menu-items @(:menu-items state)
-          :style      (style/input-view state)}
-         [rn/text-input
-          {:ref                      #(reset! (:input-ref props) %)
-           :default-value            @(:text-value state)
-           :on-focus                 #(handler/focus props state animations dimensions)
-           :on-blur                  #(handler/blur state animations dimensions subs)
-           :on-content-size-change   #(handler/content-size-change %
-                                                                   state
-                                                                   animations
-                                                                   dimensions
-                                                                   (or keyboard-shown (:edit subs)))
-           :on-scroll                #(handler/scroll % props state animations dimensions)
-           :on-change-text           #(handler/change-text % props state)
-           :on-selection-change      #(handler/selection-change % props state)
-           :on-selection             #(selection/on-selection % props state)
-           :max-height               max-height
-           :max-font-size-multiplier 1
-           :multiline                true
-           :placeholder              (i18n/label :t/type-something)
-           :placeholder-text-color   (colors/theme-colors colors/neutral-40 colors/neutral-50)
-           :style                    (style/input-text props state)
-           :max-length               constants/max-text-size
-           :accessibility-label      :chat-message-input}]]
-        [gradients/view props state animations show-bottom-gradient?]]
-       [images/images-list]
-       [actions/view props state animations window-height insets subs]]]]))
+     [reanimated/view
+      {:style     (style/sheet-container insets state animations)
+       :on-layout #(handler/layout % state blur-height)}
+      [gesture/gesture-detector
+       {:gesture (drag-gesture/drag-gesture props state animations dimensions keyboard-shown)}
+       [sub-view/bar]]
```

Comment:
Please revert this change. To make the links list scrollable use `gesture/flat-list`

Reply:
Makes sense @OmarBasem, but on the flip side, why do we need to wrap all the components in a gesture detector? I tested everything and couldn't see any problem. Seems better to wrap only the minimum amount of components inside a gesture detector.

Reply:
@ilmotta it make it easier to swipe up the composer, especially on iOS as it can read gestures even on the text input. Also not wrapping the composer would disable the one fast swipe to maximize

Reply:
I see, the fast swipe doesn't work now coming directly from the text input, only from the handle. Reverting soon, thanks @OmarBasem

Reply:
@OmarBasem, I created an issue related to the unreliability of the horizontal scroll https://github.com/status-im/status-mobile/issues/15904. It does seem like the parent gesture detector is conflicting with the `gesture/flat-list` used by the list of previews, and it stops working sometimes.

I'm taking at look at this. Do you have any clue?

Reply:
What's fascinating is that as soon as I add one image, scrolling works flawlessly, even when the input is not focused.